src/WinFormsUI/Docking/FloatWindow.cs

```diff
@@ -223,7 +220,6 @@ protected override void SetBoundsCore(int x, int y, int width, int height, Bound
             base.SetBoundsCore(x, y, width, height, specified);
         }
 
-        [SecurityPermission(SecurityAction.LinkDemand, Flags = SecurityPermissionFlag.UnmanagedCode)]

```

Comment:
Whats the reasoning behind removing this warning? It still unmanaged code and might lead to problems?

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.security.permissions.securitypermission?view=windowsdesktop-9.0&viewFallbackFrom=net-9.0&form=MG0AV3